src/main/java/ua/ithillel/lesson14/item1/SimpleCalculator.java

```diff
@@ -0,0 +1,32 @@
+package ua.ithillel.lesson14.item1;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.function.BiFunction;
+import java.util.function.Supplier;
+
+public class SimpleCalculator {
+
+    public int squareSum(int first, int second) {
+        Supplier<Integer> firstSquare = new SquareSupplier(first);
+        CompletableFuture<Integer> firstSquareFuture = CompletableFuture.supplyAsync(firstSquare);
+
+        Supplier<Integer> secondSquare = new SquareSupplier(second);
+        CompletableFuture<Integer> secondSquareFuture = CompletableFuture.supplyAsync(secondSquare);
+
+        BiFunction<Integer, Integer, Integer> combineSquares = new CombineSquares();
+        // --- получим сумму квадратов двух чисел
+        CompletableFuture<Integer> sumFuture = firstSquareFuture.thenCombine(secondSquareFuture, combineSquares);
+
+        int sum = 0;
+        try {
+            sum = sumFuture.get();
```

Comment:
```suggestion
            return sumFuture.get();
```
int sum і потім return sum - не потрібні в цьому випадку.